redshift_sqlalchemy/dialect.py

```diff
@@ -189,17 +279,106 @@ def set_isolation_level(self, connection, level):
 
         connection.set_isolation_level(level)
 
-    def _get_column_info(self, *args, **kwargs):
+    def _get_column_info(self, col, domains):
```

Comment:
@thisfred don't worry about being informal :heart: :sparkles: ! I was questioning the removal of these `*arg, **kwargs`